app/src/main/java/org/simple/clinic/bp/BloodPressureHistoryListItemDataSource.kt

```diff
@@ -9,6 +9,7 @@ import androidx.paging.toObservable
 import com.squareup.inject.assisted.Assisted
```

Comment:
I think this commit can be squashed with the earlier commit that sets this up, no?

Reply:
Yes, it's just a WIP commit I left to pick up later.